mne/io/kit/constants.py

```diff
@@ -202,6 +216,7 @@
     KIT.SYSTEM_UMD_2014_07: 'University of Maryland, 2014',
     KIT.SYSTEM_UMD_2014_12: 'University of Maryland, 2014-',
     KIT.SYSTEM_UMD_2019_09: 'University of Maryland, 2019-',
+    KIT.SYSTEM_YOKOGAWA: 'Yokogawa ??? -',
```

Comment:
@apesquita @Kyung-minAn can you help us here?

Reply:
@agramfort 
I hope that the information written above would be helpful.